lib/src/map/options/enabled_gestures.dart

```diff
@@ -0,0 +1,287 @@
+  /// if you want have no gestures enabled or only some specific gestures.
+  ///
+  /// In case you want have most or all of the gestures enabled use the
+  /// [InteractiveFlags.all] constructor instead.
+  const EnabledGestures.none({
+    this.drag = false,
+    this.flingAnimation = false,
+    this.twoFingerMove = false,
+    this.twoFingerZoom = false,
+    this.doubleTapZoomIn = false,
+    this.doubleTapDragZoom = false,
+    this.scrollWheelZoom = false,
+    this.twoFingerRotate = false,
+    this.keyTriggerDragRotate = false,
+    this.trackpadZoom = false,
+  });
+
+  /// This constructor supports bitfield operations on the static fields
+  /// from [InteractiveFlag].
+  factory EnabledGestures.bitfield(int flags) {
+    return EnabledGestures(
+      drag: InteractiveFlag.hasFlag(flags, InteractiveFlag.drag),
+      flingAnimation:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.flingAnimation),
+      twoFingerMove:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerMove),
+      twoFingerZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerZoom),
+      doubleTapZoomIn:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.doubleTapZoomIn),
+      doubleTapDragZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.doubleTapDragZoom),
+      scrollWheelZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.scrollWheelZoom),
+      twoFingerRotate:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.twoFingerRotate),
+      keyTriggerDragRotate:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.keyTriggerDragRotate),
+      trackpadZoom:
+          InteractiveFlag.hasFlag(flags, InteractiveFlag.trackpadZoom),
+    );
+  }
+
+  /// Enable panning with a single finger or cursor
+  final bool drag;
+
+  /// Enable fling animation after panning if velocity is great enough.
+  final bool flingAnimation;
+
+  /// Enable panning with multiple fingers
+  final bool twoFingerMove;
+
+  /// Enable zooming with a multi-finger pinch gesture
+  final bool twoFingerZoom;
+
+  /// Enable rotation with two-finger twist gesture
+  final bool twoFingerRotate;
+
+  /// Enable zooming with a single-finger double tap gesture
+  final bool doubleTapZoomIn;
+
+  /// Enable zooming with a single-finger double-tap-drag gesture
+  ///
+  /// The associated [MapEventSource] is [MapEventSource.doubleTapHold].
+  final bool doubleTapDragZoom;
+
+  /// Enable zooming with a mouse scroll wheel
+  final bool scrollWheelZoom;
+
+  /// Enable zooming with the device trackpad / touchpad
+  final bool trackpadZoom;
+
+  /// Enable rotation by pressing the defined keyboard key (by default CTRL key)
+  /// and dragging with the cursor
+  /// or finger.
+  final bool keyTriggerDragRotate;
+
+  /// Returns true of any gesture with more than one finger is enabled.
+  bool hasMultiFinger() => twoFingerMove || twoFingerZoom || twoFingerRotate;
```

Comment:
Prefer a getter here? Maybe under another name?

```dart
  bool get hasMultiFinger => twoFingerMove || twoFingerZoom || twoFingerRotate;
```

_(`suggestion` not made to prevent introduction of bugs elsewhere)_

Reply:
Removed this method because it was only used in `MapInteractiveViewer.updateGestures` and could cause confusion between multi point touchscreen and mutli point trackpad gestures.